backend/api/Database/Models/UserInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#pragma warning disable CS8618
+namespace Api.Database.Models
+{
+    public class UserInfo
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
+        [Required]
+        public string Username { get; set; }
```

Comment:
We can maybe have a discussion later whether the username is needed here, but for now we could maybe leave it like this, assuming we address it before moving it to production.